docs/USAGE.md

```diff
@@ -16,7 +16,7 @@ java -jar gtfs-validator-v2.0.jar --input relative/path/to/dataset.zip --output
 ...which will:
  1. Search for a GTFS dataset located at `relative/path/to/dataset.zip`
```

Comment:
We should itemize all the possible command-line parameters in this file. If the README has a partial list that's fine, as long as there is a reference pointing to this file for the full parameter list.

Reply:
Above reference to Circle CI in line 6 should be removed.

Reply:
Also on line 6, are snapshot artifacts still being stored?